Regression/Checksum/benchmarks_json/Python_prev_positions.json

```diff
@@ -6,9 +6,9 @@
     "Ex": 3710588.849989976,
     "Ey": 0.0,
     "Ez": 646727.8074440088,
-    "jx": 4745.078379617619,
```

Comment:
These changes are surprisingly large! It is interesting to see how much of a difference the two methods have when noise is very high.